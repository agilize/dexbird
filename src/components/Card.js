import React from 'react'
import PropTypes from 'prop-types';

export const Card = ({ title, children }) => (
  <div className="card">
    <div className="card-header">{ title }</div>
    <div className="card-body">{ children }</div>
  </div>
);

Card.propTypes = {
  title: PropTypes.string.isRequired,
  children: PropTypes.any.isRequired,
};

export default Card;
