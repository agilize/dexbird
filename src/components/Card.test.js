import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';
import Card from './Card';

it('renders without crashing', () => {
  const wrapper = shallow(<Card title="Balance">10</Card>);
  expect(wrapper).toMatchSnapshot();
});
