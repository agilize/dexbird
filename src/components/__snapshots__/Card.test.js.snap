// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="card"
>
  <div
    className="card-header"
  >
    Balance
  </div>
  <div
    className="card-body"
  >
    10
  </div>
</div>
`;
